pkg/security/apis/security/install/install.go

```diff
@@ -13,8 +13,8 @@ import (
 )
 
 func init() {
-	Install(legacyscheme.GroupFactoryRegistry, legacyscheme.Registry, legacyscheme.Scheme)
 	legacy.InstallLegacySecurity(legacyscheme.Scheme, legacyscheme.Registry)
+	Install(legacyscheme.GroupFactoryRegistry, legacyscheme.Registry, legacyscheme.Scheme)
```

Comment:
why this switch?

Reply:
> why this switch?

Last one wins. I found it in a separate PR and this is a good spot to switch.  All the rest are already like this.